tests/io/datasets/test_frms6.py

```diff
@@ -197,50 +296,48 @@ def test_read_invalid_tileshape(default_frms6):
         next(p.get_tiles(tiling_scheme=tiling_scheme))
 
 
-def test_positive_sync_offset(lt_ctx):
+def test_positive_sync_offset(default_frms6_raw, lt_ctx):
```

Comment:
Can we make the test that depends on the raw version independent of the other? Then we get at least some coverage if stemtool is not installed.

Reply:
Ok, I've made the verification packages optional and skip only the tests that require it if they are not available.